apps/public-reference/pages/applications/review/summary.tsx

```diff
@@ -101,144 +48,7 @@ export default () => {
-                  <br />
-                  {application.alternateContact.phoneNumber}
-                  <br />
-                  {application.alternateContact.emailAddress}
-                </ReviewItem>
-              </div>
-            </>
-          )}
-
-        <h3 className="form--card__sub-header">
-          {t("application.household.householdMembers")}
-          <EditLink href="/applications/household/add-members" />
-        </h3>
-
-        {application.householdSize > 0 && (
-          <div className="form-card__group info-group mx-0">
-            {application.householdMembers.map((member) => (
-              <div className="info-group__item" key={`${member.firstName} - ${member.lastName}`}>
-                <p className="info-item__value">
-                  {member.firstName} {member.lastName}
-                </p>
-                <div>
-                  <ReviewItem label={t("application.household.member.dateOfBirth")}>
-                    {member.birthMonth}/{member.birthDay}/{member.birthYear}
-                  </ReviewItem>
-                  {member.sameAddress === "no" && (
-                    <ReviewItem label={t("application.contact.address")}>
-                      <MultiLineAddress address={reformatAddress(member.address)} />
-                    </ReviewItem>
-                  )}
-                  {member.sameAddress !== "no" && (
-                    <ReviewItem label={t("application.review.sameAddressAsApplicant")}></ReviewItem>
-                  )}
-                </div>
-              </div>
-            ))}
-          </div>
-        )}
-        {application.householdSize === 0 && (
-          <div className="form-card__group mx-0">{t("application.review.noAdditionalMembers")}</div>
-        )}
-
-        <h3 className="form--card__sub-header">
-          {t("application.review.householdDetails")}
-          <EditLink href="/applications/household/ada" />
-        </h3>
-
-        <div className="form-card__group mx-0">
-          <ReviewItem label={t("application.ada.label")}>
-            {accessibilityLabels(application.accessibility).map((item) => (
-              <Fragment key={item}>
-                {item}
-                <br />
-              </Fragment>
-            ))}
-          </ReviewItem>
-        </div>
-
-        <h3 className="form--card__sub-header">
-          {t("t.income")}
-          <EditLink href="/applications/financial/vouchers" />
-        </h3>
-
-        <div className="form-card__group border-b mx-0">
-          <ReviewItem label={t("application.review.voucherOrSubsidy")}>
-            {application.incomeVouchers ? t("t.yes") : t("t.no")}
-          </ReviewItem>
-
-          <ReviewItem label={t("t.income")}>
-            ${application.income} {t(`application.financial.income.${application.incomePeriod}`)}
-          </ReviewItem>
-        </div>
+        <FormSummaryDetails application={application} editMode />
 
         <h3 className="form--card__sub-header">
```

Comment:
The new Preferences section should probably get moved into the FormSummaryDetails component as well.

Reply:
Absolutely yes! I think I missed this element after the merge.